scripts/create-triggers-accounts.sql

```diff
@@ -0,0 +1,61 @@
+USE fxa;
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+-- Clear any existing triggers
```

Comment:
A bit of info about what the `_accounts_new` table is actually used for would be useful.

Reply:
I added some comments in a subsequent commit that I'll squash later.